src/main/resources/application.yml

```diff
@@ -15,4 +15,8 @@ org:
       logging:
         LoggingSystem: none
 jar:
-  path: 
\ No newline at end of file
+  path:
+# TODO Actual values for domain and url need to be provided
```

Comment:
We can't have default values for this so I guess they should be empty. The users need to specify it in their config file.